R/print_interval.R

```diff
@@ -113,7 +113,7 @@ print_interval.numeric <- function(
       interval <- paste0(
           conf_level
           , "$["
-          , paste(strip_math_tags(x), collapse = ", ")
+          , paste(strip_math_tags(x), collapse = ",~")
```

Comment:
I don't like the idea of using a non-breaking space here. I think a line break between lower and upper boundary of a CI is much better than a line break between, for instance
```
p <
.001
```
Adding a non-breaking space in the CI increases the probability that nasty edge cases like the one I showed here occurs, so I would keep the old behavior. If we would really want to be consistent with that, `p~<~.001` would be of much higher priority than the two values of the CI, I suppose.

Reply:
Good point. Maybe we can use `\ ` instead.

Reply:
Wouldn't that still prioritize keeping the interval together compared with the space between other symbols?

Reply:
> Wouldn't that still prioritize keeping the interval together

I'm not really sure, but it should not be a non-breaking space. I would have to try it out. Also, I'm not sure what a "space between other symbols" is, but if you mean `[x.xx, x.xx]`: The space in equation will not be rendered, which is why I considered `~` in the first place.

Reply:
What I mean with "space between other symbols" are instances such as the one I described here with the *p* value. If we have a results section with a couple of in-line equations, it is likely that a CI is reported in close proximity to such a p value. If LaTeX would have to decide whether to add a line break within the CI or within the "p = .xxx", it would be forced to add the line break within the "p = .xxx". I think this is undesirable, and we should therefore not add a non-breaking space within the cI.